packages/fastboot/src/scripts/install-source-map-support.js

```diff
@@ -1,12 +1,10 @@
 'use strict';
 /* globals sourceMapSupport */
 
-Error.prepareStackTrace = function prepareStackTrace(error, stack) {
-  return error + stack.map(frame => '\n    at ' + sourceMapSupport.wrapCallSite(frame)).join('');
-};
```

Comment:
Note: this is removed because `source-map-support` never uses the pre-existing `Error.prepareStackTrace` value ([it always clobbers](https://github.com/evanw/node-source-map-support/blob/ac2c3e4c633c66931981ac94b44e6963addbe3f4/source-map-support.js#L581-L585)).